extension/test/integration/make-unauthorized-payment.handler.spec.js

```diff
@@ -0,0 +1,107 @@
+const { expect } = require('chai')
+
+const ctpClientBuilder = require('../../src/ctp')
+const config = require('../../src/config/config')
+const constants = require('../../src/config/constants')
+const iTSetUp = require('./integration-test-set-up')
+
+describe('::unauthorized make-payment request use case::', () => {
```

Comment:
What about a test for an authorized make-payment use case?

Reply:
Modified the integration test
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/c241b45b82d627a1fd1e33de861c8f9efff435e2

For easier review on the changes, i would modify the filename in another commit.